scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError
+            HTTP, GraphQL, UserError, CheckoutError
```

Comment:
I'm not sure about calling this `CheckoutError`. I don't really want to use `ApplePayError` either since thats too platform specific. Maybe `PaymentError`?